### PR TITLE
Override top & bottom margin of Coursetalk widget

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd 

https://trello.com/c/GHu2XyZW/140-enabling-coursetalk-widget-instance-wide-on-sandbox-works-but-might-need-some-css-layout-improvement

### CODE REVIEW

**Comment by caesar2164** [comment]
[Enabling CourseTalk widget instance-wide on sandbox works but might need some CSS / layout improvement (?)](https://trello.com/c/GHu2XyZW/140-enabling-coursetalk-widget-instance-wide-on-sandbox-works-but-might-need-some-css-layout-improvement)

**Comment by stvstnfrd** [comment]
What about CME and SUclass?

**Comment by caesar2164** [comment]
they don't use coursetalk

**Comment by stvstnfrd** [comment]
But is there anything about it that precludes them from doing so?
If CME wanted to start using it, it would be broken.
If we spin up a new deployment/theme in the future, it would also be broken.

Is the Lagunita theme itself explicitly breaking this styling,
or is it an issue w/ the platform's styling itself?

If it's the latter, we shouldn't be patching just one theme.

File: lagunita/lms/static/sass/_overrides.scss [hunk 1]
```diff
@@ -238,8 +238,14 @@ body.discussion {
...
 }
 
-.course-sidebar .research-info {
-    padding: 0 10px;
+.course-sidebar {
+    .research-info {
+        padding: 0 10px;
+    }
+    >.coursetalk-read-reviews {
+        margin-top: 50px !important;
```
Comment (stvstnfrd) [inline lagunita/lms/static/sass/_overrides.scss]: Let's use more specific selectors instead of `!important`.

**Comment by caesar2164** [comment]
This is actually a platform issue, so closing this PR in favor of https://github.com/Stanford-Online/edx-platform/pull/700

**Pull request closed without merging**